src/Cake.Common/Build/AzurePipelines/Data/AzurePipelinesPublishCodeCoverageData.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.IO;
+
+namespace Cake.Common.Build.AzurePipelines.Data
+{
+    /// <summary>
+    /// Providers TF Build agent for publishing code coverage results
```

Comment:
Comment doesn't make lot of sense. Suggestion:

`Description of code coverage information to publish to Azure Pipelines`